app/controllers/organizations_controller.rb

```diff
@@ -0,0 +1,14 @@
+class OrganizationsController < ApplicationController
+
+  def create
+    set_current_user
```

Comment:
With the above change to `current_user`, the method doesn't need to be called ahead of time, instead of using `@current_user` on line 7, you can just use `current_user` the method.